src/pages/jumbotron.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import JumbotronC from '../components/pages/home/jumbotron'
+import {GetServerSideProps} from 'next'
+import {loadCourse} from '../lib/courses'
+import groq from 'groq'
+import {sanityClient} from '../utils/sanity-client'
+
+export const Jumbotron = ({resource}: {resource: any}) => {
+  console.log(resource)
+  return (
+    <main className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
+      <div className="max-w-screen-xl mx-auto">
+        <div className="lg:space-y-6 space-y-4">
+          <JumbotronC resource={resource} />
+        </div>
+      </div>
+    </main>
+  )
+}
+
+export default Jumbotron
+
+export const getServerSideProps: GetServerSideProps = async ({res, params}) => {
+  const load = async () => {
+    const data = await sanityClient.fetch(groq`
```

Comment:
this query is a lot more digestible

right now we need to do a lot of transformation from the Sanity content model to our local `resource`. this could be simplified considerably if we changed what local components expect.